harness/tests/experiment/pytorch/test_pytorch_context.py

```diff
@@ -0,0 +1,50 @@
+import unittest
+
+import torch
+
+from determined import pytorch
+from determined.common.check import CheckFailedError
+from determined.errors import InternalException
+from tests.experiment.fixtures import pytorch_onevar_model
+
+
+class TestPyTorchContext(unittest.TestCase):
+    def setUp(self) -> None:
```

Comment:
I'm assuming that `setUp` gets called automatically by pytest somehow?  I couldn't find any docs about it, so I wanted to confirm.

Reply:
I believe it does, depending on `TestPyTorchContext` inheriting from `unittest.TestCase`. I know our other other test classes don't do this, but I wanted to take advantage of the `assertRaises` method.

Reply:
Ah, I missed the TestCase subclassing part.  That explains my confusion.

Can you use `with pytest.raises` instead?  It has the same functionality but it matches the rest of our test code.

Reply:
This is done.